lms/static/coffee/src/instructor_dashboard/send_email.coffee

```diff
@@ -34,6 +34,7 @@ class SendEmail
     @$email_messages_wrapper = @$container.find(".email-messages-wrapper")
 
     # attach click handlers
+    accessible_modal(".keyword-sub-link", "#keyword-sub-modal .close-modal", "#keyword-sub-modal", "#section-send-email")
```

Comment:
This could be like this I think:
accessible_modal(".keyword-sub-link", ".keyword-sub-modal .close-modal", ".keyword-sub-modal", "#section-send-email")